src/Vehicle/Vehicle.cc

```diff
@@ -3290,13 +3331,10 @@ void Vehicle::setSoloFirmware(bool soloFirmware)
     }
 }
 
-#if 0
-// Temporarily removed, waiting for new command implementation
-void Vehicle::motorTest(int motor, int percent, int timeoutSecs)
+void Vehicle::motorTest(int motor, int percent)
 {
-    doCommandLongUnverified(_defaultComponentId, MAV_CMD_DO_MOTOR_TEST, motor, MOTOR_TEST_THROTTLE_PERCENT, percent, timeoutSecs);
+    sendMavCommand(_defaultComponentId, MAV_CMD_DO_MOTOR_TEST, true, motor, MOTOR_TEST_THROTTLE_PERCENT, percent, 0, 0, MOTOR_TEST_ORDER_BOARD);
```

Comment:
So is MOTOR_TEST_ORDER_BOARD the real connection order as opposed to the strange Mission Planner motor order that ArduPilot is using. Did you implement normal motor number ordering support in ArduPilot or only for ArduSub?

Reply:
For example for multi-rotor motor numbering?

Reply:
The reason I ask is because this is in the generic code. For example when PX4 implement motor test (using normal motor numbering) would this be the setting for them as well. Hence it is correct for all firmwares.

Reply:
Yeah, this is the real order.
IIRC @jaxxzer got this implemented into ArduSub only.

Reply:
I'm open to suggestions here.
We could expose the order to be used
`motorTest(int motor, int percent, MOTOR_TEST_ORDER order)`

Reply:
> Yeah, this is the real order.

This is fine then